IS/Sem.07/Solutions.cpp

```diff
@@ -0,0 +1,135 @@
+#include <iostream>
+
+size_t myStrLen(const char* str) {
+    size_t index = 0;
+
+    while (str[index] != '\0') {
+        index++;
+    }
+
+    return index;
+}
+
+int myStrCmp(const char* str1, const char* str2) {
+    size_t i = 0;
+
+    while (str1[i] == str2[i]) {
+        if (str1[i] == '\0'/* && str2[i] == '\0'*/) {
+            return 0;
+        }
+
+        i++;
+    }
+
+    return str1[i] - str2[i] > 0 ? 1 : -1;
```

Comment:
Ternary is not necessary.
Just return str1[i] - str2[i]

Reply:
The students wanted it with -1 and 1 :D